### PR TITLE
will update ticket 5 with functionality and error tests

### PR DESCRIPTION
**Pull request by MilesPerAlwin**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -80,4 +80,55 @@ describe("CORE: GET /api/articles/:article_id error test suite", () => {
...
     })
+});
+describe("CORE: GET /api/articles functionality test suite", () => {
+    test("returns a 200 GET request with an article array of article objects without a body property in date descending order", () => {   
+        return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles).toHaveLength(13);
+            const isSorted = body.articles.every((val, i, arr) => !i || arr[i-1].created_at >= val.created_at);
```
Comment (RichardNeat) [inline __tests__/app.test.js:91]: This is a very inventive line of code! If you wanted to explore a nicer way of handling this, the npm package jest sorted can do just this for you :)
https://www.npmjs.com/package/jest-sorted

File: controllers/articles-controller.js [hunk 2]
```diff
@@ -9,4 +9,13 @@ exports.getArticleById = (req, res, next) => {
...
         next(err);
     })
-}
\ No newline at end of file
+}
+
+exports.getArticles = (req, res, next) => {
+    selectArticles()
+    .then((articlesArr) => {
+        // currently returning articles array in DESC order, but no comment_count
```
Comment (RichardNeat) [inline controllers/articles-controller.js]: Please remove any comments and console logs before committing :)

**Review by RichardNeat — APPROVED** [review]
Nice work Alwin! Just a couple of suggestions in the comments below but you're good to merge :)
P.S. also remember to update your json endpoints file as you go 👍

File: controllers/articles-controller.js (hunk 2 above)
Comment (MilesPerAlwin) [reply controllers/articles-controller.js]: Woops I missed this, thank you! Also when I went to the endpoint.json I noticed that there was already an entry for GET /api/articles - do I have to write another one, or tweak this one? Yesterday eve I saw it and didn't read all of it, now I can see the description is slightly different than the functionality I've made.

Comment (RichardNeat) [reply controllers/articles-controller.js]: Good spot - yeah might just need some tweaking :)

**Pull request merged**